examples/cord_epsim/main.c

```diff
@@ -26,6 +26,7 @@
 #include "net/cord/common.h"
 #include "net/sock/util.h"
 #include "net/ipv6/addr.h"
+#include "xtimer.h"
```

Comment:
Why add xtimer here if the PR is about removing xtimer?

Reply:
this example is using xtimer but didn't include it (was used via gcoap.h)

Reply:
Ok, won't block it in any case, I would have just have gone the other way around

Reply:
Ok, won't block it in any case, I would have just have gone the other way around